### ISSUE TITLE
Add small notes to Candidate Widget

### ISSUE THREAD

**Issue by gl1000007** [hasadna/open-shot#395]
![zadik](https://f.cloud.github.com/assets/5072931/1044435/a3902690-1011-11e3-9af4-15bc5e4660b4.PNG)
**Low priority** + only after #388 and #390 are done.
This issue is relevant to the first 2 lines - yellow and white notes.

**Comment by gl1000007** [comment]
design issue, leave it for the next elections.
